dev/com.ibm.ws.cdi.internal/src/com/ibm/ws/cdi/internal/archive/AbstractCDIArchive.java

```diff
@@ -70,8 +72,8 @@ public AbstractCDIArchive(String name, CDIRuntime cdiRuntime) {
      * <p>
      * If a class name cannot be found, it is ignored.
      *
-     * @param archive the BDA
-     * @param internalClassNames classes to load only if they're loaded directly by the BDA's classloader
+     * @param archive              the BDA
```

Comment:
Did you get a tab here?

Reply:
No, it's just that the auto-formatter thinks the descriptions should be indented so that they start on the same column. This file hasn't changed for ages so it was probably out of date compared to how files are formatted now.